libs/build.gradle

```diff
@@ -7,7 +7,33 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-configure(childProjects.values() - project('elasticsearch-log4j')) {
+configure(childProjects.values()) {
+
+  apply plugin: 'base'
+
+  def baseProject = project
+  def baseName = "elasticsearch-${it.name}"
```

Comment:
The term "base name" doesn't really make sense here anymore. Can we call this `artifactId` instead? I think that describes what where' trying to influence here.